server.js

```diff
@@ -56,6 +60,13 @@ if (plugins.legacyGovukFrontendFixesNeeded()) {
 app.locals.pluginConfig = plugins.getAppConfig({
   scripts: utils.prototypeAppScripts
 })
+
+// Set up GOV.UK Frontend relative paths
+app.locals.govukFrontend = {
+  assetPath: govukFrontend.assetPath,
+  scripts: govukFrontend.config?.scripts || []
+}
```

Comment:
@nataliecarey I think these lines need a suggestion

Nunjucks views will get `govukFrontend` set to the internal GOV.UK Frontend version

Is there a router/middleware approach we can use to set:

1. Set Nunjucks local `govukFrontend` to "internal version" by default
2. Set Nunjucks local `govukFrontend` to "plugin version" in plugin pages only

Reply:
Turns out Nunjucks, Sass and the error server had overlapping GOV.UK Frontend versions:

* https://github.com/alphagov/govuk-prototype-kit/pull/2333

But now they're separated I've added separate locals for each:

```mjs
// Add GOV.UK Frontend paths to Nunjucks locals
app.locals.govukFrontend = govukFrontend
app.locals.govukFrontendInternal = govukFrontendInternal
```

This allows prototype pages and management pages to clearly use their own versions:

```njk
{%- set assetPath = '/manage-prototype/dependencies/govuk-frontend' + govukFrontendInternal.assetPath -%}
```

Or for **govuk-branded.njk** it can be overridden or default to the plugin version:

```njk
{%- set assetPath = assetPath | default('/plugin-assets/govuk-frontend' + govukFrontend.assetPath) -%}
```